azkaban-common/src/main/java/azkaban/utils/AbstractMailer.java

```diff
@@ -32,20 +31,13 @@
   private final String azkabanName;
   private final String tls;
 
-  private final long attachmentMazSizeInByte;
-
   public AbstractMailer(final Props props) {
     this.azkabanName = props.getString("azkaban.name", "azkaban");
     this.mailHost = props.getString("mail.host", "localhost");
     this.mailPort = props.getInt("mail.port", DEFAULT_SMTP_PORT);
     this.mailUser = props.getString("mail.user", "");
     this.mailPassword = props.getString("mail.password", "");
     this.tls = props.getString("mail.tls", "false");
-    final long maxAttachmentSizeInMB =
-        props.getInt("mail.max.attachment.size.mb", 100);
```

Comment:
Can't find references to this property anywhere in the repo either. Seems like it's been undocumented (kinda expected).

Reply:
This may be used for limiting the attachment size for reportal. Did you check the plugin repo?

Reply:
In the plugin repo there's `ReportalMailCreator.java` but it implements the `MailCreator` interface [directly](https://github.com/azkaban/azkaban-plugins/blob/84ab2772b4680f0f10e3d04d666465d34b09a0d9/plugins/reportal/src/azkaban/viewer/reportal/ReportalMailCreator.java#L52).

It has its own [MAX_ATTACHMENT_SIZE](https://github.com/azkaban/azkaban-plugins/blob/84ab2772b4680f0f10e3d04d666465d34b09a0d9/plugins/reportal/src/azkaban/viewer/reportal/ReportalMailCreator.java#L62) constant, it doesn't refer to the code I'm deleting on this side.

Reply:
Thanks for checking!